mk/config.mk

```diff
@@ -57,7 +66,9 @@ endif #config
 .PHONY: configs
 configs:
 ifeq ($(shell realpath $(CONFIG_DIR)),$(shell realpath $(CONFIGS_SUBMODULE_DIR)))
-	git submodule update --init --remote -- $(CONFIGS_SUBMODULE_DIR)
+	@echo "Updating config submodule: $(CONFIGS_SUBMODULE_DIR)"
```

Comment:
This won't work as intended in BF. It will checkout config version matching main repository commit, not newest one

Reply:
It will work for master only. revisions/PRs etc will use version from when they branched off from master. It is IMO correct behavior (it uses tested, matching config), but consensus was to use actual config version.

Reply:
Moving to correct behaviour... expecting to solve issue with config changes appearing in peoples PRs